ontology/ontoavida.md

```diff
@@ -30,3 +30,9 @@ tags:
 tracker: https://gitlab.com/fortunalab/ontoavida/-/issues
 activity_status: active
 ---
+<img  src="https://fortunalab.org/images/alife_bacteria.jpg" height="150px" align="left"/>The **Ontology for Avida ([OntoAvida](https://gitlab.com/fortunalab/ontoavida))** project aims to develop an integrated vocabulary for the description of [Avida](https://github.com/devosoft/avida), the most widely used computational approach for performing experimental evolution. The lack of a clearly defined vocabulary makes biologists feel reluctant to embrace the field of digital evolution. This unique ontology has the potential to change this picture overnight. In addition, OntoAvida will allow researchers to make inference (e.g., on phenotypic plasticity) based on certain rules and constraints, facilitate the reproducibility of the *in silico* evolution experiments reported in the [scientific literature](https://gitlab.com/fortunalab/ontoavida#references), and trace the provenance of the data stored in AvidaDB, a semantic database that stores genomes and transcriptomes of more than a million digital organisms.
+
+# Products
+
+- [ontoavida.owl](https://gitlab.com/fortunalab/ontoavida/-/raw/master/ontoavida.owl)
+- [ontoavida.obo](https://gitlab.com/fortunalab/ontoavida/-/raw/master/ontoavida.obo)
```

Comment:
Why list the products again? They are already listed:

<img width="789" alt="image" src="https://user-images.githubusercontent.com/7070631/186374759-25ca8155-5e35-41bf-8ff2-91e9267fab14.png">

Reply:
Dear @matentzn, It is a mistake from my side, I apologize. Removing. Thanks